src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -45,11 +45,6 @@ async function boot(userOptions?: Partial<BlazorOptions>): Promise<void> {
   const circuitHandlers: CircuitHandler[] = [new AutoReconnectCircuitHandler(logger)];
   window['Blazor'].circuitHandlers = circuitHandlers;
 
-  // In the background, start loading the boot config and any embedded resources
-  const embeddedResourcesPromise = fetchBootConfigAsync().then(bootConfig => {
-    return loadEmbeddedResourcesAsync(bootConfig);
-  });
```

Comment:
There's more involved in removing this feature:

 * We should also remove the equivalent code from `Boot.WebAssembly.ts`
 * We should also remove any underlying infrastructure that used to support these old-style embedded resources, e.g., other MSBuild targets
 * We should update the templates and E2E tests that rely on it

Altogether my guess is you won't want to do all that as part of this PR. If that's correct, maybe we could leave out the changes in this file and remove all this stuff one time later.

Reply:
I would like to start getting rid of this on the server-side part to make sure it doesn't make it to 3.0. I'll file a bug to finish cleaning-up the blazor client-side pieces of it.

Reply:
OK cool!